.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,39 @@
+name: "Feature request"
+description: Suggest an idea for the Python devguide
+title: "Feature: <title>"
+labels: ["enhancement"]
+assignees: []
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > [!NOTE]
+        > This repo is for the Python devguide. If you are requesting an
+        > enhancement for the Python language or CPython interpreter,
+        > then the CPython issue tracker is better
+        > suited for this report: https://github.com/python/cpython/issues
+
+  - type: textarea
+    id: feature_description
+    attributes:
+      label: "Describe the enhancement or feature you'd like"
```

```suggestion
      label: "Describe the enhancement or feature you would like"
```
Abbreviations should be avoided.